### ISSUE TITLE
Unable to archive Google Sheet

### ISSUE THREAD

**Issue by edsu** [webrecorder/archiveweb.page#174]
When I tried to archive this public Google Sheet:

https://docs.google.com/spreadsheets/d/1dKmaV_JiWcG8XBoRgP8b4e9Eopkpgt7FL7nyspvzAsE/edit#gid=0

I got an error "Aw Snap something went wrong when displaying this web page". It looks like the page got partially archived, but it fails to play back because it pops up a warning about a "Server Error" and then another error that requires reloading the page.

https://github.com/webrecorder/archiveweb.page/assets/33829/955b4da6-60a5-428c-8dbe-01d02b364d46

The console has some errors about `This document requires 'TrustedScriptURL' assignment.` which looks like it might be significant?

```
__awp_main_inject__:23 This document requires 'TrustedScriptURL' assignment.
(anonymous) @ __awp_main_inject__:23
__awp_main_inject__:23    Uncaught TypeError: Failed to set the 'src' property on 'HTMLScriptElement': This document requires 'TrustedScriptURL' assignment.
    at __awp_main_inject__:23:15
(anonymous) @ __awp_main_inject__:23
2228254062-waffle_js_prod_core.js:1986 Uncaught TypeError: Cannot read properties of null (reading 'error')
    at qbc.aa (2228254062-waffle_js_prod_core.js:1986:36)
    at u.log (2228254062-waffle_js_prod_core.js:1991:396)
    at Ebe.window.onerror (2228254062-waffle_js_prod_core.js:5987:458)
qbc.aa @ 2228254062-waffle_js_prod_core.js:1986
u.log @ 2228254062-waffle_js_prod_core.js:1991
Ebe.window.onerror @ 2228254062-waffle_js_prod_core.js:5987
__awp_main_inject__:1 Object
__awp_main_inject__:23 This document requires 'TrustedScriptURL' assignment.
(anonymous) @ __awp_main_inject__:23
__awp_main_inject__:23    Uncaught TypeError: Failed to set the 'src' property on 'HTMLScriptElement': This document requires 'TrustedScriptURL' assignment.
    at __awp_main_inject__:23:15
    at new LJd (2228254062-waffle_js_prod_core.js:4847:23)
    at new TJd (2228254062-waffle_js_prod_core.js:4873:158)
    at u.rTd (2228254062-waffle_js_prod_core.js:6022:365)
    at hs (2228254062-waffle_js_prod_core.js:917:292)
    at 2228254062-waffle_js_prod_core.js:5966:367
    at hs (2228254062-waffle_js_prod_core.js:917:292)
    at ube (2228254062-waffle_js_prod_core.js:6003:392)
    at 2228254062-waffle_js_prod_core.js:5964:204
    at hs (2228254062-waffle_js_prod_core.js:917:292)
(anonymous) @ 2228254062-waffle_js_prod_core.js:1993
DevTools failed to load source map: Could not load content for https://docs.google.com/spreadsheets/d/1dKmaV_JiWcG8XBoRgP8b4e9Eopkpgt7FL7nyspvzAsE/fontloading_binary_fontloading_binary_chunk.sourcemap: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://docs.google.com/spreadsheets/d/1dKmaV_JiWcG8XBoRgP8b4e9Eopkpgt7FL7nyspvzAsE/inline_binary_core.sourcemap: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://docs.google.com/static/spreadsheets2/client/js/waffle_js_prod_core.sourcemap: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
__awp_main_inject__:1 {data: 'Loaded behaviors for: about:blank', type: 'debug'}
__awp_main_inject__:23 This document requires 'TrustedScriptURL' assignment.
(anonymous) @ __awp_main_inject__:23
...
```

**Comment by ikreymer** [comment]
Thanks for including the video, the initial issue seems to be same as #160 and will have a fix for it in upcoming 0.11.1 release.

However, even with the recording working, there may be a separate replay issue which may require some time to debug :/

**Comment by edsu** [comment]
Thanks the fix for #160 did the trick. I will ticket the replay issue separately.
